packages/core/src/internal-types.ts

```diff
@@ -18,6 +18,7 @@ export interface ParsedRequest extends ServerRequest {
 }
 
 export type FindRoute = (request: ParsedRequest) => ResolvedRoute<string>;
+export type Handler = (request: ServerRequest, response: ServerResponse) => Promise<void>;
```

Comment:
Since this handler is specific to HTTP(s) requests, I am proposing to rename it to `HttpHandler`